spring-cloud-dataflow-rest-client/src/main/java/org/springframework/cloud/dataflow/rest/client/TaskTemplate.java

```diff
@@ -134,51 +136,51 @@ public class TaskTemplate implements TaskOperations {
 				EXECUTION_RELATION_BY_NAME,
 				EXECUTIONS_INFO_RELATION,
 				PLATFORM_LIST_RELATION,
-				RETRIEVE_LOG
+				RETRIEVE_LOG,
+				VALIDATION_REL
```

Comment:
What is this now?

Reply:
This was rolled back in the version I want to push.